.github/workflows/requirements.txt

```diff
@@ -1,4 +1,5 @@
 mkdocs==1.2.3
 mkdocs-simple-hooks==0.1.3
 mkdocs-material==7.1.9
-mike==1.0.1
\ No newline at end of file
+mike==1.0.1
+git+git://github.com/pygments/pygments@38cd147859fdaeb73a0a2018601f4ec5277798bc#egg=pygments
```

Comment:
that's until they release right ?

Reply:
yes - but last release was in august so I don't expect it anytime soon. I will watch and update to latest release as soon as I observe it

Reply:
but I just noticed an error in the PR

Reply:
fixed 😅 .... just to make sure the lib is actually used with the specific commit version